.github/workflows/update-axe-core.yml

```diff
@@ -0,0 +1,37 @@
+name: Update axe-core
+
+on: [workflow_dispatch]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      - id: update
+        uses: dequelabs/update-axe-core@main
```

Comment:
Are we pinning to main?

Reply:
I don't see why not. I can create a `v1` branch if you'd prefer though.